github/apps.go

```diff
@@ -32,8 +36,21 @@ type App struct {
 
 // InstallationToken represents an installation token.
 type InstallationToken struct {
-	Token     *string    `json:"token,omitempty"`
-	ExpiresAt *time.Time `json:"expires_at,omitempty"`
+	Token        *string                  `json:"token,omitempty"`
+	ExpiresAt    *time.Time               `json:"expires_at,omitempty"`
+	Permissions  *InstallationPermissions `json:"permissions,omitempty"`
+	Repositories *[]Repository            `json:"repositories,emitempty"`
+}
+
+// InstallationTokenParameters allow restricting a token's access to specific repositories.
+type InstallationTokenParameters struct {
+	// The ids of the repositories that the installation token can access.
+	// Providing repository ids restricts the access of an installation token to specific repositories.
```

Comment:
Let's please change `ids` in the comment to `IDs` for the auto-generated godocs.

Reply:
Done.